addons/web/static/src/js/core/owl_popover.js

```diff
@@ -0,0 +1,257 @@
+                    name: 'left',
+                    top: targetTop - (popoverHeight - targetHeight) / 2,
+                    left: targetLeft - popoverWidth,
+                },
+            };
+        }
+
+        /**
+         * Check if the element is in the viewport.
+         * @private
+         * @static
+         * @param {HTMLElement}
+         * @returns {Boolean} True if the element currently fits inside 
+         *                    the viewport, false otherwise.
+         */
+        static _isInViewport(element) {
+            const rect = element.getBoundingClientRect();
+            const html = document.documentElement;
+            return (
+                rect.top >= 0 &&
+                rect.left >= 0 &&
+                rect.bottom <= (window.innerHeight || html.clientHeight) &&
+                rect.right <= (window.innerWidth || html.clientWidth)
+            );
+        }
+
+        /**
+         * Hide any displayed popover, display the popover in argument and mark
+         * it as the one being displayed in the class attribute. Add an event
+         * listener on the document to detect any click outside the popover as a
+         * closing event.
+         * Note that the event listener is added for the capture phase, meaning
+         * that it will *always* run before the click listener set on a popover
+         * template. This means that there can be no race condition regarding
+         * clicking on another popover trigger: any existing popover will be
+         * removed during the capture phase, then a new listener will be added
+         * in the bubbling phase of the new popover' trigger click event.
+         * @static
+         * @param {Popover} popover The popover component to display.
+         */
+        static display(popover) {
+            /* this should never happen because of the way the events are handled,
+            but in case an imaginative dev puts a popover inside another one, i'd
+            prefer the first one to be hidden anyway, causing the second one to
+            never display itself (since its target won't be in the viewport) */
+            if (this.displayed) {
+                this.displayed._close();
+            }
+            if (!this.isListening) {
+                document.addEventListener('click', Popover.documentClickHandler, { capture: true });
+                this.isListening = true;
+            }
+            this.displayed = popover;
+            popover.state.displayed = true;
+        }
+
+        /**
+         * Hide any displayed popover and remove event listeners.
+         */
+        static hide() {
+            const popover = this.displayed;
+            document.removeEventListener('click', Popover.documentClickHandler, { capture: true });
+            this.isListening = false;
+            popover.state.displayed = false;
+            this.displayed = null;
+        }
+    }
+
+    const documentClickHandler = function (e) {
```

Comment:
I tried in my console, and since static variables are not supported, I guess there is no choice but to do what you've done....